src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -775,7 +775,7 @@ private static int GetMatchingNamedConstructorArgumentIndex(string parameterName
             return argIndex;
         }
 
-        private TypedConstant GetDefaultValueArgument(ParameterSymbol parameter, AttributeSyntax syntax, BindingDiagnosticBag diagnostics)
+        private TypedConstant GetDefaultValueArgument(ParameterSymbol parameter, AttributeSyntax syntax, ImmutableArray<string> constructorArgumentNamesOpt, int argumentsCount, BindingDiagnosticBag diagnostics)
```

Comment:
>GetDefaultValueArgument [](http://example.com/codeflow?start=30&length=23)

We have two call sites and only one test scenario. It feels like one call site doesn't have test coverage

Reply:
@AlekseyTs I added few more tests. But seems I'm still hitting one call site only.

Reply:
> I added few more tests. But seems I'm still hitting one call site only. 

Is this still the case or were you able to cover both code paths?

Reply:
@AlekseyTs It's not the case now.